### PR TITLE
Refactor MTE-1208 [v123] navigation tests with TR

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1208)

## :bulb: Description
Data alignment for navigation tests with TR.  Below screenshots results are for testScrollsToTopWithMultipleTabs that has been modified to use the newly added function scrollToElement.
https://docs.google.com/document/d/14nhRcn4PtHFo1VMGh9w6FPpyOUjgElw751JU0yuuuw0/edit#heading=h.i26ccowgusqi
https://testrail.stage.mozaws.net/index.php?/suites/view/45443&group_by=cases:section_id&group_order=asc&group_id=426835
Screenshot 2024-01-03 at 09 17 15
Screenshot 2024-01-03 at 09 17 33

### CODE REVIEW

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift [hunk 1]
```diff
@@ -403,7 +413,7 @@ class NavigationTest: BaseTestCase {
         XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
         app.typeText("example.com\n")
 
-//        waitUntilPageLoad()
+        // waitUntilPageLoad()
```
Comment (isabelrios) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift:416]: Nit: can we remove this instead of leaving it commented?

**Review by isabelrios — APPROVED** [review]
Just a small comment, otherwise LGTM

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift (hunk 1 above)
Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift:416]: I can update this in the next PR if that is ok with you.

Comment (isabelrios) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift:416]: sure, no problem

**Pull request merged**